### DISCUSSION TITLE
Nginx dropping connections

### DISCUSSION THREAD

**Discussion by vimalmistry** [uNetworking/uWebSockets.js#584] [General]
I am using nginx for reverse proxy .

```
server{
	server_name  echo.orspot.com;
       	location / {
                         server_tokens off;
  		   	proxy_pass http://localhost:9001;
			proxy_http_version 1.1;
			proxy_set_header Upgrade $http_upgrade;
			proxy_set_header Connection 'upgrade';
			proxy_set_header Host $host;
			proxy_cache_bypass $http_upgrade;
		}
}

```

 uWebsocket.js keep dropping connection. I found out it nginx problem. because working ok on my local machine.

Any Idea?

**Comment by e3dio** [comment]
Is it every 60 seconds? see https://github.com/uNetworking/uWebSockets.js/issues/519#issuecomment-831223172

**Reply by vimalmistry** [reply]
Yes. 60 second. I calculated :)
